A quest/order update

Arodtanjoe opened the pull request:
Updated the AQUest file to add more items for AQ high, and a few others.   This will help add more variety to the AQ high space.  Additional items added for Veteran and Uber orders as well.

KParzival reviewed (commented):
Have a few concerns here - particularly adding the difficult elementals to the mob AQ list, there are several classes that would struggle with that and find it very frustrating.

I'm unaware of why Tarion Cavalier was removed, I thought he was a good 6 pter long ago, same with Shomed.

As for the objects, they are mostly fine (sword made from bone is a great add for example), but I think we might need to take a deeper look at repop value on a couple as it relates to their point value before pushing to live.


Additional notes (2025-01-26):
Guard of the Gates (26700) - 4
Rhoden (11701) - 5
uncomment Tarion Cavalier (6)
uncomment  Shomed (5)
ancient flame dragon (11703) - 6
Cerberus (28705) - 6
Charon (28704) - 6
Emma (14210) - 6
A Deep Wyrm (5556) - 6
Jason (14201) - 6
The Enlightened One (26584) - 6

If we want to get fancy - adjust specs for Emir, Pirate Captain, Typik, Neyuv, Reptilian Demon to not do their throw-from-room spec when below 20% hp.

Arodtanjoe commented:
> Have a few concerns here - particularly adding the difficult elementals to the mob AQ list, there are several classes that would struggle with that and find it very frustrating.
> 
> I'm unaware of why Tarion Cavalier was removed, I thought he was a good 6 pter long ago, same with Shomed.
> 
> As for the objects, they are mostly fine (sword made from bone is a great add for example), but I think we might need to take a deeper look at repop value on a couple as it relates to their point value before pushing to live.
> 
> Additional notes (2025-01-26): Guard of the Gates (26700) - 4 Rhoden (11701) - 5 uncomment Tarion Cavalier (6) uncomment Shomed (5) ancient flame dragon (11703) - 6 Cerberus (28705) - 6 Charon (28704) - 6 Emma (14210) - 6 A Deep Wyrm (5556) - 6 An Eight-Legged Drider (5523) - 5 Jason (14201) - 6 The Enlightened One (26584) - 6
> 
> Comment out: { 5105, 5 }, /* drow apprentice */
> 
> If we want to get fancy - adjust specs for Emir, Pirate Captain, Typik, Neyuv, Reptilian Demon to not do their throw-from-room spec when below 20% hp.

1/28/2025

AQUEST Updates


Hachiman -  A Gleaming Jade Battlesuit - 6 -> 12
Hachiman - Helm of Lacquered Wood 8 -> 11
Haciman - Yawata's Ring of Power - 5 -> 12


Emith - A Clean Read Cloak - 5 -> 9
Emith - A Glowing Staff of Gnarled Wood - 5 -> 10

Ogres -  a swarm of angry beetles - 4 -> 9
OGres - a mass of black, glistening thorns  - 5 -> 10
Ogres - A Rotting Otyugh Skin  - 2 -> 8

Daimyo - Icon of Possession  - 4 -> 8
Daimyo - Ebon Armguards - 5 -> 9


Tregannis - A Band of Demonic Aura - 5-> 9
Tregannis - The Mallet of Orcus - 6-> 9

Wax - the Dagger of Segretto - 5 -> 9



If we want to get fancy - adjust specs for Emir, Pirate Captain, Typik, Neyuv, Reptilian Demon to not do their throw-from-room spec when below 20% hp.



Emir: Already accounts for 20%
Pirate Captain - DOne
Typik - Done
Neyuv - Throws when greater than 10% - Will adjust to 20
Reptillian Demon - Updated

Arodtanjoe commented:
The latest set of commits updates a few additional zone's and their item repop rates.  It will make it more inline with the rest of the Aquest orders.   Also updated the AQuest mob list and adjusted some point values.   This is to add more variety to both quests and orders.

danewoodall commented on src/aquest.c:
Is it possible that we could handle typos for aq orders? I'm not sure the logic, but you get defaulted to a low, or maybe even random order type, if the one you request isn't expected, i.e. low/high/uber/etc.

For instance, if you 'aq order hihg', you should be given a message from Franz saying maybe he doesn't understand, rather than giving you a random/low order.

KParzival replied:
I went ahead and put this in https://github.com/RoninMUD/ronin/pull/23

KParzival commented:
We currently have a workflow for spec changes but not lib/world changes (pending) - I suggest you move all the hp_threshold throw changes into the BM disarm PR (https://github.com/RoninMUD/ronin/pull/12), that way all the same type of changes are in the same PR.

KParzival commented:
I think it could speed up this review/implementation process if you threw out the annoying files that get added due to zone edits.  For example, lib/world/BelowtheSewersbyInfinity.wld has no changes, other than the annoying fact that editing the zone does some weird ordering of descs and so it acts like its new.  Remove all instances of that kind of thing and it gives a reviewer (like me) a lot less meaningless drek to sift through.
